src/OrbitGl/ThreadTrack.h

```diff
@@ -66,9 +66,9 @@ class ThreadTrack final : public TimerTrack {
   void UpdatePositionOfSubtracks();
   void UpdateMinMaxTimestamps();
 
-  std::shared_ptr<orbit_gl::ThreadStateTrack> thread_state_track_;
-  std::shared_ptr<orbit_gl::EventTrack> event_track_;
-  std::shared_ptr<orbit_gl::TracepointTrack> tracepoint_track_;
+  std::shared_ptr<orbit_gl::ThreadStateBar> thread_state_track_;
+  std::shared_ptr<orbit_gl::CallstackThreadBar> event_track_;
+  std::shared_ptr<orbit_gl::TracepointThreadBar> tracepoint_track_;
```

Comment:
I guess that these should be renamed.

Reply:
Good catch, I missed those... changed, thanks!